registration/migrations/0006_auto_20171225_0938.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-12-25 09:38
+from __future__ import unicode_literals
+
+from django.core import validators
+from django.db import migrations, models
+from django.db.models import Q, Func, F
+
+import registration.models
+
+
+def migrate_to_lowercase_domains(apps, schema_editor):
```

Comment:
Could you please split this migration into two parts, one which adds the subdomain validation to the model, and a separate data migration for fixing the existing internal domains?

Then, if you submit the data change as a separate PR, then we can merge the schema change and deploy it, and at least stop any new invalid domains from being created until we can sort out the manual changes required.